src/components/WorldMap.js

```diff
@@ -1,12 +1,120 @@
 import "./styles/WorldMap.css";
 import { Link } from "react-router-dom";
+import { useState, useEffect } from "react";
+import ReactMapGl, { Marker } from "react-map-gl";
 
 export default function WorldMap() {
+  const [countries, setCountries] = useState([]);
+  const [visitedCountries, setVisitedCountries] = useState(Number(0));
```

Comment:
You don't need to use `Number(0)` as the initial state, as 0 is already a number. You can just do `useState(0)`.
You only need to convert to `Number` if what you are passing is not a number already. e.g. `Number("0")`